src/pages/components/notification/accessibility.mdx

```diff
@@ -0,0 +1,228 @@
+---
+title: Notification
+description: Notifications are messages that communicate information to the user. The two main types of notificaitons are toast notifications and inline notifications.
+tabs: ['Code', 'Usage', 'Style', 'Accessibility']
+---
+
+import {
+  StructuredListWrapper,
+  StructuredListHead,
+  StructuredListBody,
+  StructuredListRow,
+  StructuredListInput,
+  StructuredListCell,
+  OrderedList,
+  ListItem,
+} from 'carbon-components-react';
+
+<AnchorLinks>
+  <AnchorLink>How it works</AnchorLink>
+  <AnchorLink>Accessibility considerations</AnchorLink>
+  <AnchorLink>Resources</AnchorLink>
+  <AnchorLink>Accessibility testing</AnchorLink>
+</AnchorLinks>
+
+## How it works
+
+Notification Carbon components are messages that communicate information to a user. The WAI-ARIA `role="alert"` is equivalent to `aria-live="assertive"`. It is used to display a message to the user in a way that attracts the user's attention without receiving focus to communicate the message without interrupting the users current task. If the notification does not contain an urgent message, consider changing it to `aria-live="polite"` so the notification is queued until whatever task the user is currently working on is complete.
+
+The close button contains an ARIA label of `aria-label="close"`, and the icon has `aria-hidden="true"` so it is ignored by assistive technologies. The `Tab` key is used to move focus to the close button within the notification and the `Space` key or `Enter` key activate the button to close the notification.
+
+## Accessibility considerations
+
+1. Avoid using a toast notifiction for critical alerts or long messages because they are timed and will disappear automatically making it difficult for people with various disabilities to get the entire message. An alert that disappears too quickly can lead to failure of the optional [WCAG 2.0 success criterion 2.2.3 (AAA)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-no-exceptions.html).
+2. Provide a means to make nonessential alerts optional to enhance usability for people with visual and cognitive disabilities. which also make meeting the requirements of the optional [WCAG 2.0 success criterion 2.2.4 (AAA)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-postponed.html) more difficult.
```

Comment:
```suggestion
2. Provide a means to make nonessential alerts optional to enhance usability for people with visual and cognitive disabilities. which also make meeting the requirements of the optional [WCAG 2.0 success criterion 2.2.4 (AAA)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-postponed.html) more difficult.
```

This isn't a complete sentence, though I'm not sure what it's trying to say.

Reply:
@snidersd do you have any thoughts on how we can change this sentence?

Reply:
@aledavila Here is a rewrite:
Provide a means to turn off nonessential alerts to enhance usability for people with visual and cognitive disabilities. Additional information is available in the [WCAG 2.0 success criterion 2.2.4 (AAA)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-postponed.html)success criterion. Note: Providing this functionality is optional.